src/App.css

```diff
@@ -11,39 +11,93 @@ body {
   margin: 0;
   padding: 0;
   height: 100%;
+  background-color: lightyellow;
 }
 
 #root {
   height: 100%;
 }
 
-.main {
+.navbar {
   display: flex;
-  justify-content: space-around;
-  flex-wrap: wrap;
-  padding: 2rem;
+  justify-content: space-between;
+  padding-inline: 10%;
+}
+
+.navbar h1 {
+  font-size: 3rem;
+  color: goldenrod;
+}
+
+.navbar ul {
+  list-style-type: none;
+  display: flex;
+  gap: 5%;
+  width: 100%;
+  font-size: 1.5rem;
+  position: relative;
+  top: 1rem;
+}
+
+.nav-link {
+  text-decoration: none;
+}
+
+/* ---------------Home Page------------------ */
+
+.homepage {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] KIndly make sure to align the homepage contents in order to match the provided design and project requirement.

  ### YOUR DESIGN

   ![chrome-capture-2023-7-22](https://github.com/AliRezaBmeDu/math-magician/assets/87197412/41fbf376-1601-454e-8507-1910eb2831af)


  ### PROVIDED DESIGN

    ![chrome-capture-2023-7-22 (1)](https://github.com/AliRezaBmeDu/math-magician/assets/87197412/0da19024-e72c-40ca-8e8e-314f0415b31d)